examples/platforms/da15000/tool.h

```diff
@@ -0,0 +1,162 @@
+/*
```

Comment:
These macros are overkill for this example.  Many of these are not used.  Most are unnecessary.  See the cc2538 example.